db/migrate/20210930162822_add_audited_user_and_username_to_audits.rb

```diff
@@ -0,0 +1,6 @@
+class AddAuditedUserAndUsernameToAudits < ActiveRecord::Migration[6.1]
+  def change
+    add_column :audits, :audited_user_id, :integer
+    add_column :audits, :audited_username, :string
```

Comment:
Is this used for tracking the user that made a change or the user that was changed? For the former, I think we already have `audit.user_id` and for the latter we already have `audit.auditable_id` when `audit.auditable_type == 'User'`

Reply:
This is for tracking the actual user that was changed, auditable tracks `ProjectMember` for example if you add a user to a project, not User. This extracts the actual user and stores that and provides a common way to access it. The audited_user_id and audited_username helps significantly when it comes to tracking deleted project members since `auditable_id` no longer provides any useful data when the associated `ProjectMember` has been deleted.

Reply:
sounds good thanks for the explanation